bika/lims/locales/es/LC_MESSAGES/bika.po

```diff
@@ -5915,28 +6470,48 @@ msgid "The laboratory departments"
 msgstr ""
 
 #: bika/lims/content/sampletype.py:115
-msgid "The list of sample points from which this sample type can be collected.  If no sample points are selected, then all sample points are available."
-msgstr "La lista de puntos de muestreo de desde poden ser recogidas las muestras de este tipo. Si no selecciona ningún punto de muestreo, entonces estarán disponibles todos los puntos de muestreo."
+msgid ""
+"The list of sample points from which this sample type can be collected.  If "
+"no sample points are selected, then all sample points are available."
+msgstr ""
+"La lista de puntos de muestreo de desde poden ser recogidas las muestras de "
```

Comment:
@xispa _muestreo donde se pueden recoger_ instead of _muestreo de desde poden ser recogidas_